src/dsp_tools/commands/excel2json/properties.py

```diff
@@ -226,32 +229,33 @@ def _get_final_series(
     return final_series
 
 
-def _row2prop(df_row: pd.Series[Any], row_num: int, excelfile: str) -> dict[str, Any]:
-    _property = {x: df_row[x] for x in mandatory_properties} | {
-        "labels": get_labels(df_row=df_row),
-        "super": [s.strip() for s in df_row["super"].split(",")],
-    }
-    if not pd.isna(df_row["subject"]):
-        _property["subject"] = df_row["subject"]
-
+def _row2prop(df_row: pd.Series[Any], row_num: int, excelfile: str) -> Property:
+    subj = df_row["subject"] if not pd.isna(df_row["subject"]) else None
+    comment = get_comments(df_row=df_row)
     gui_attrib = _get_gui_attribute(df_row=df_row, row_num=row_num)
     match gui_attrib:
-        case dict():
-            _property["gui_attributes"] = gui_attrib
         case InvalidExcelContentProblem():
             msg = f"There is a problem with the excel file: '{excelfile}'\n" + gui_attrib.execute_error_protocol()
             raise InputError(msg) from None
-
-    if comment := get_comments(df_row=df_row):
-        _property["comments"] = comment
-
-    return _property
+        case _:
+            pass
```

Comment:
The match case doesn't make sense any more, because you only want to take action in 1 case. Then, a simple `if` is more appropriate (and uses only 3 LOC instead of 6 LOC :-)

```
if isinstance(gui_attrib, InvalidExcelContentProblem):
        msg = f"There is a problem with the excel file: '{excelfile}'\n" + gui_attrib.execute_error_protocol()
        raise InputError(msg)
```

Reply:
In addition: `raise Exception() from None`: the `from None` is only meaningful if you are inside an `except` block. Here, it's superfluous.